extension-files/tools/zabbix_discovery_devices

```diff
@@ -157,10 +157,17 @@ parser.add_argument(
 
 parser.add_argument(
     '--only_rawdisk',
-    help='Discovery for real raw/hardware devices',
+    help='Discovery for real raw/hardware devices and partitions',
```

Comment:
The `and partitions` in the Help does not match the be behaviour in https://github.com/zabbix-tooling/zabbix-agent-extensions/pull/51/commits/50eec03db54ccaaff5749513d9431d1ad56078c1#diff-f22d4c12383ab2ad75b41e7720997c17699c17e4237ce0ed82022005ecdd2fd2R274

Reply:
Tnx, adapted the description.